atomicapp/cli/main.py

```diff
@@ -52,9 +52,10 @@ def print_app_location(app_path):
 
 def cli_genanswers(args):
     argdict = args.__dict__
+    location = argdict['answers']
```

Comment:
I still don't think we need to do this here. What do you think about just passing it in with the rest of the args and do it inside the function in main.py ?